website/static/css/pages/remove-contributor-page.css

```diff
@@ -0,0 +1,4 @@
+.col-md-4 {
```

Comment:
Seems dangerous to override a Bootstrap class like this; is there a better way to do this?